build-system/server/new-server/transforms/transform.ts

```diff
@@ -19,15 +19,32 @@ import minimist from 'minimist';
 import posthtml from 'posthtml';
 import transformModules from './modules/modules-transform';
 import transformScriptPaths from './scripts/scripts-transform';
-import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
 
 const argv = minimist(process.argv.slice(2));
-const transforms = [transformStories(), transformScriptPaths(), transformCss()];
+const FOR_TESTING = argv._.includes('integration');
```

Comment:
Curious if this is different from `const FOR_TESTING = !!argv.integration`

Reply:
it looks like it is as the task name is not a key on the argv object when i run `gulp integration...`
<img width="925" alt="Screen Shot 2020-10-06 at 9 49 54 AM" src="https://user-images.githubusercontent.com/354746/95232436-587f1600-07b9-11eb-88ed-2af9b80b84b6.png">